tests/unittests/OperatorTest.cpp

```diff
@@ -3665,6 +3665,44 @@ TEST_P(InterpOnly, rowwiseQuantizedFCTest) {
   }
 }
 
+TEST_P(Operator, SoftMax) {
```

Comment:
nit: Comments here and for the other test?

Reply:
Sure thing!